pkg/kubectl/resource_printer.go

```diff
@@ -1749,6 +1749,9 @@ func printRoleBindingList(list *rbac.RoleBindingList, w io.Writer, options Print
 }
 
 func printClusterRole(clusterRole *rbac.ClusterRole, w io.Writer, options PrintOptions) error {
+	if options.WithNamespace {
+		return fmt.Errorf("clusterRole is not namespaced")
```

Comment:
This doesn't tell the user what they did wrong.  Can you update the message to be more clear about what the user shoudl do?